extensions/spark/kyuubi-spark-lineage/src/main/scala/org/apache/kyuubi/plugin/lineage/dispatcher/atlas/AtlasEntityHelper.scala

```diff
@@ -104,33 +104,38 @@ object AtlasEntityHelper {
   }
 
   def tableObjectId(tableName: String): Option[AtlasObjectId] = {
-    val dbTb = tableName.split('.')
-    if (dbTb.length == 2) {
-      val qualifiedName = tableQualifiedName(cluster, dbTb(0), dbTb(1))
-      // TODO parse datasource type
-      Some(new AtlasObjectId(HIVE_TABLE_TYPE, "qualifiedName", qualifiedName))
-    } else {
-      None
+    tableName.split('.') match {
+      case Array(catalog, db, table) =>
```

Comment:
We should ignore the catalog for Atlas, because the Hive Hook in Atlas also does not specify a catalog:

https://github.com/apache/atlas/blob/b7849aeb78856f2f8e205c5e784cee84409cbc72/addons/hive-bridge/src/main/java/org/apache/atlas/hive/hook/events/BaseHiveEvent.java#L884

Reply:
we can only handle the default `spark_catalog` catalog.

Reply:
The implementation here is to merge the catalog into the table name or column name, and the result will be in the form of a string. Does that matter?

Reply:
I think the catalog will only be used to judge the data source type and not be part of the qualifiedName.